sbin/Makefile.am

```diff
@@ -55,12 +28,33 @@ if ENABLE_VNETBUILD
 scripts += vnetbuild
 endif
 
-sbin_SCRIPTS = $(scripts)
+CLEANFILES = install.config
 
-EXTRA_DIST = \
-	commands.sed.in \
-	$(scriptsin) \
-	$(inclib_DATA)
+inclib_DATA = \
+	functions.common \
+	install.config \
+	$(NULL)
+
+inclib_SCRIPTS = $(scripts)
 
-uninstall-local:
+EXTRA_DIST = \
+	firehol.in \
+	fireqos.in \
+	link-balancer.in \
+	update-ipsets.in \
+	vnetbuild.in \
+	functions.common.in \
+	install.config.in \
+	$(NULL)
```

Comment:
usually, you do not need to extra dist the autoconf templates, automake does it for you.